python/graphstorm/run/gsgnn_emb/gsgnn_node_emb.py

```diff
@@ -63,36 +70,50 @@ def main(config_args):
         "restore model path cannot be none for gs_gen_node_embeddings"
 
     # load the model
-    if config.task_type == BUILTIN_TASK_LINK_PREDICTION:
-        model = gs.create_builtin_lp_gnn_model(input_data.g, config, train_task=False)
-    elif config.task_type in {BUILTIN_TASK_NODE_REGRESSION, BUILTIN_TASK_NODE_CLASSIFICATION}:
-        model = gs.create_builtin_node_gnn_model(input_data.g, config, train_task=False)
-    elif config.task_type in {BUILTIN_TASK_EDGE_CLASSIFICATION, BUILTIN_TASK_EDGE_REGRESSION}:
-        model = gs.create_builtin_edge_gnn_model(input_data.g, config, train_task=False)
+    if config.multi_tasks:
+        # Only support multi-task sahred encoder model.
+        model = GSgnnMultiTaskSharedEncoderModel(config.alpha_l2norm)
+        gs.gsf.set_encoder(model, input_data.g, config, train_task=False)
```

Comment:
can we use `gs.set_encoder` here?

Reply:
set_encoder is not included in gsf.__init__.